modules/programs/taskwarrior.nix

```diff
@@ -98,9 +88,12 @@ in {
   config = mkIf cfg.enable {
     home.packages = [ pkgs.taskwarrior ];
 
-    home.file.".taskrc".text = ''
+    xdg.configFile."task/taskrc".text = ''
       data.location=${cfg.dataLocation}
-      ${includeTheme cfg.colorTheme}
+      ${optionalString (cfg.colorTheme != null) (if isString cfg.colorTheme then
+        "include ${cfg.colorTheme}.theme"
```

Comment:
Since the suffix `.theme` is automatically added now, wouldn't this break some user configurations?

Reply:
We used to also automatically add `.theme`.
https://github.com/nix-community/home-manager/pull/2455/files#diff-5055038a8f44cd12c4b9b48ebd41a9611a3243b83b890fef804756052968ca6fL9

Reply:
Right, my mistake